lapce-ui/src/settings.rs

```diff
@@ -357,13 +357,15 @@ impl Widget<LapceTabData> for LapceSettingsPanel {
                     .get_color_unchecked(LapceTheme::EDITOR_BACKGROUND),
             );
 
-            for (i, text) in ["Core Settings", "Editor Settings", "Keybindings"]
+            static SETTINGS_SECTIONS: [&str; 3] = ["Core Settings", "Editor Settings", "Keybindings"];
+
+            for (i, text) in SETTINGS_SECTIONS
                 .iter()
                 .enumerate()
             {
                 let text_layout = ctx
                     .text()
-                    .new_text_layout(text.to_string())
+                    .new_text_layout(*text)
```

Comment:
I might be wrong, but I think calling `into_iter()` on the array (which should be a `const`, really) would allow removing this dereference.

Reply:
Done.